src/main/java/dev/openfga/sdk/api/client/OpenFgaClient.java

```diff
@@ -1104,6 +1104,167 @@ public CompletableFuture<ClientListObjectsResponse> listObjects(
         return call(() -> api.listObjects(storeId, body, overrides)).thenApply(ClientListObjectsResponse::new);
     }
 
+    /**
+     * StreamedListObjects - Stream all objects of a particular type that the user has a relation to.
+     * This method provides true asynchronous streaming with consumer callbacks.
+     * Objects are delivered to the consumer as they are received from the server asynchronously.
+     * Returns a CompletableFuture that completes when streaming is finished.
+     *
+     * <p>Example usage:</p>
+     * <pre>{@code
+     * ClientListObjectsRequest request = new ClientListObjectsRequest()
+     *     .user("user:anne")
+     *     .relation("viewer")
+     *     .type("document");
+     *
+     * client.streamedListObjects(request,
+     *     response -> System.out.println("Found object: " + response.getObject())
+     * ).thenRun(() -> System.out.println("Streaming complete"))
+     *  .exceptionally(error -> {
+     *      System.err.println("Error: " + error.getMessage());
+     *      return null;
+     *  });
+     * }</pre>
+     *
+     * @param request The list objects request containing type, relation, and user
+     * @param consumer Callback to handle each StreamedListObjectsResponse as it arrives
+     * @return CompletableFuture<Void> that completes when streaming finishes
+     * @throws FgaInvalidParameterException When the Store ID is null, empty, or whitespace, or consumer is null
+     */
+    public CompletableFuture<Void> streamedListObjects(
+            ClientListObjectsRequest request, Consumer<StreamedListObjectsResponse> consumer)
```

Comment:
Typically we return client wrappers over the API return type, which include http info like status, headers. Obviously in this case we are streaming so it's different. Do other SDKs return generated type directly, or provide a client wrapper?